File: 1043.py
x = input().split()
a, b, c =x
a = float(a)
b = float(b)
c = float(c)

if abs(b - c) < a < (b + c) and (a - c) < b < (a + c) and (a - b) < c < (a + b):
    print('Perimetro = {:.1f}'.format(a + b + c))
else:
    print('Area = {:.1f}'.format(((a + b) / 2) * c))

File: C#/1043.cs
using System;
using System.Linq;
using System.Globalization;

class Program {
    static void Main() 
    {
        decimal[] lista  = Console.ReadLine().Split().Select(decimal.Parse).ToArray();
        if(lista[0]< lista[1]+ lista[2] && lista[1]<lista[0]+lista[2] && lista[2]< lista[0]+lista[1])
        {
            Console.WriteLine($"Perimetro = {lista[0]+lista[1]+lista[2]}");
        }
        else{
            Console.WriteLine($"Area = {(lista[0]+lista[1])*lista[2]/2:F1}");
        }
    }
}
       
   